### PR TITLE
Feature/cypress circle ci

### PR DESCRIPTION
**Pull request by peggles2**
This pull requests updates circle ci config to run cypress tests.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v2-deps-{{ .Branch }}-{{ checksum "pnpm-lock.yaml" }}
-            - v2-deps-{{ .Branch }}-
-            - v2-deps-
+            - v1-deps-{{ .Branch }}-{{ checksum "pnpm-lock.yaml" }}
+            - v1-deps-{{ .Branch }}-
+            - v1-deps-
```
Comment (kscoulter) [inline .circleci/config.yml:17]: Did you have to downgrade the version to run Cypress?

**Review by kscoulter — COMMENTED** [review]
Did you have to register the cypress run? I got confused at that step, where you have to set up a dashboard.

File: .circleci/config.yml (hunk 1 above)
Comment (peggles2) [reply .circleci/config.yml:17]: Not ready for code review yet.  That really doesn't matter its just clearing the cache. I'll let you know when ready for review.

Comment (kscoulter) [reply .circleci/config.yml]: Ah ok :)

**Review by kscoulter — DISMISSED** [review]
Great solution!

**Pull request merged**